js/electron.js

```diff
@@ -38,6 +38,8 @@ function createWindow() {
 		electronOptions.autoHideMenuBar = true;
 	}
 
+	var electronOptions = Object.assign({}, electronOptionsDefaults, config.electronOptions);
```

Comment:
This is broken, because the above if-else sets properties on `electronOptions`, but only `electronOptionsDefaults` and `config.electronOptions` are merged here.